docs/data/material/getting-started/templates/dashboard/components/SelectContent.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import {
+  Avatar as MuiAvatar,
+  ListItemAvatar as MuiListItemAvatar,
+} from '@mui/material';
+import MenuItem from '@mui/material/MenuItem';
+import ListItemText from '@mui/material/ListItemText';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListSubheader from '@mui/material/ListSubheader';
+import Select, { SelectChangeEvent, selectClasses } from '@mui/material/Select';
+import Divider from '@mui/material/Divider';
+import { styled } from '@mui/material/styles';
+import AddRoundedIcon from '@mui/icons-material/AddRounded';
+import DevicesRoundedIcon from '@mui/icons-material/DevicesRounded';
+import SmartphoneRoundedIcon from '@mui/icons-material/SmartphoneRounded';
+import ConstructionRoundedIcon from '@mui/icons-material/ConstructionRounded';
+
+const Avatar = styled(MuiAvatar)(({ theme }) => ({
+  width: 28,
+  height: 28,
+  backgroundColor: theme.palette.background.paper,
+  color: theme.palette.text.secondary,
+  border: `1px solid ${theme.palette.divider}`,
+}));
+
+const ListItemAvatar = styled(MuiListItemAvatar)({
+  minWidth: 0,
+  marginRight: 12,
+});
+
+export default function SelectContent() {
+  const [company, setCompany] = React.useState('');
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setCompany(event.target.value as string);
+  };
+
+  return (
+    <Select
+      labelId="company-select"
+      id="company-simple-select"
+      value={company}
+      onChange={handleChange}
+      displayEmpty
+      inputProps={{ 'aria-label': 'Select company' }}
+      fullWidth
+      sx={{
+        maxHeight: 56,
+        width: 215,
+        '&.MuiList-root': {
```

Comment:
We should use `listClasses.root`

Reply:
Tried replacing it for:

```
        [`&. ${listClasses.root}`]: {
          p: '8px',
        },
```

and importing it like:
`import listClasses from '@mui/material/List';`

However, I got the error: 
`Property 'root' does not exist on type 'ExtendList<ListTypeMap<{}, "ul">>'.ts(2339) `